warp10/src/main/java/io/warp10/continuum/egress/Egress.java

```diff
@@ -107,6 +108,7 @@ public Egress(KeyStore keystore, Properties props, boolean fetcher) throws Excep
 
     if (useHttp) {
       int port = Integer.valueOf(props.getProperty(Configuration.EGRESS_PORT));
+      int tcpBacklog = Integer.valueOf(props.getProperty(Configuration.EGRESS_TCP_BACKLOG, "50"));
```

Comment:
Jetty's default value is 0, meaning use the system default, it would be better to have the same behavior here instead of the arbitrary value 50.